pkg/gce-pd-csi-driver/controller.go

```diff
@@ -145,7 +145,14 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 			return nil, status.Error(codes.AlreadyExists, fmt.Sprintf("CreateVolume disk already exists with same name and is incompatible: %v", err))
 		}
 		// If there is no validation error, immediately return success
-		return generateCreateVolumeResponse(existingDisk.GetSelfLink(), capBytes, zones), nil
+		return generateCreateVolumeResponse(existingDisk.GetSelfLink(), capBytes, zones, existingDisk.GetSourceSnapshot()), nil
+	}
+	snapshotId := ""
```

Comment:
nit: add a space before this line to seperate

Reply:
fixed.